### ISSUE TITLE
1.4.4 Resize Text: EN 301 549 did not incorporate our updated notes

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#704]
The EN 301 549 has different notes from WCAG2ICT for 1.4.4 Text Spacing. The reasoning given:
>Note 2 is an exception and also contradicts our current notes.   If the platform doesn’t do it – the software is supposed to.

## 2013 WCAG2ICT
2013 WCAG2ICT Notes say:
>Note 1: [Content](https://www.w3.org/WAI/GL/wcag2ict/#keyterms_content) for which there are software players, viewers or editors with a 200 percent zoom feature would automatically meet this success criterion when used with such players, unless the content will not work with zoom.
>
>Note 2: The Intent section refers to the ability to allow users to enlarge the text on screen at least up to 200% without needing to use [assistive technologies](https://www.w3.org/WAI/GL/wcag2ict/#atdef). This means that the application provides some means for enlarging the text 200% (zoom or otherwise) without loss of [content](https://www.w3.org/WAI/GL/wcag2ict/#keyterms_content) or functionality or that the application works with the platform features **that meet this requirement**.

## 2024 WCAG2ICT
Current WCAG2ICT Notes say:
>Note 1 (Added): **The Intent section refers to** the ability to allow users to enlarge the text on screen at least up to 200% without needing to use [assistive technologies](https://w3c.github.io/wcag2ict/#dfn-assistive-technologies). This means that the application provides some means for enlarging the text 200% (zoom or otherwise) without loss of [content](https://w3c.github.io/wcag2ict/#content-on-and-off-the-web) or functionality, or that the application works with the platform **accessibility** features **to satisfy this success criterion**.
>
>Note 2 (Added): **For non-web software, there may be cases where the platform does not scale all text up to 200%. In such cases, authors are encouraged to meet user needs by scaling text to the extent supported by user settings in the platform.**

## EN 301 549
The EN 301 549 Notes use the 2013 WCAG2ICT Note 1 for both non-web documents and software, create a new Note 2 for non-web documents, and modified the first phrase of our Note 1 (2013 WCAG2ICT's Note 2):

**Clause 10.1.4.4 for non-web documents**
>NOTE 1:  Content for which there are software players, viewers or editors with a 200 percent zoom feature would automatically meet this success criterion when used with such players, unless the content will not work with zoom.
>
>NOTE 2:  **It is best practice to use only fonts that allow for scaling without loss of quality (e.g. pixelized presentation). This applies in particular to embedded fonts.**

**Clause 11.1.4.4 for non-web software**
>NOTE 1: Content for which there are **non-web** software players, viewers or editors with a 200 percent zoom feature would automatically meet this success criterion when used with such players, unless the content will not work with zoom.
>
>NOTE 2: **This success criterion is about** the ability to allow users to enlarge the text on screen at least up to 200 % without needing to use assistive technologies. This means that the application provides some means for enlarging the text 200 % (zoom or otherwise) without loss of content or functionality, or that the application works with the platform features **that meet this requirement**.

-----------------
## Decisions to make
For non-web software, we need to determine:
1. If we need to revert the WCAG2ICT notes back to the 2013 version for SC 1.4.4 (which basically removes the current WCAG2ICT Note 2 and add back in the 2013 Note 1).
2. If we need to incorporate the change to our current Note 1 to replace the phrase "The Intent section refers to the ability..." to "This success criterion is about the ability...".
3. Should we replace "the application" in our current note 1 with "the non-web software".

For non-web documents, we need to determine:
1. If the notes in the current WCAG2ICT are appropriate for non-web docs OR
2. If the notes in the EN 301 549 are more appropriate (uses Note 1 from 2013 WCAG2ICT and adds a note about using fonts that allow for scaling.

**Comment by mapluke** [comment]
I too think that Loïc has it just right.

**Comment by ChrisLoiselle** [comment]
Following this conversation, Loic's response seems to be the way to go. I look forward to our conversation on Thursday to discuss further.

**Comment by maryjom** [comment]
@detlevhfischer It does, so we are actually working on adjusting the SC language for non-web software. We're working in a [Google doc for 1.4.4](https://docs.google.com/document/d/1VN1LQG_iI6ySWSc-XBgfE6d_aHQRUy_CjMGktPdFenM/edit?usp=sharing) on possible language changes so software programs, especially in mobile, aren't always responsible for testing the platform features to make sure everything scales to 200% and then being required to make up for the shortfalls where the platforms settings don't cause text to scale 200%.  IMO, there should be appropriate platform requirements in place, and software should simply support the user's text size choices in the platform settings.

**Comment by maryjom** [comment]
The TF reached resolutions on the various parts to the 1.4.4 guidance in the 31 July meeting on the content changes and PR 748 has been merged into the editor's draft. The decisions are documented on the [TF decisions wiki page](https://github.com/w3c/wcag2ict/wiki/WCAG2ICT-Task-Force-Decisions).
